src/typings/tacoLiveReload.d.ts

```diff
@@ -53,6 +53,7 @@ declare module TacoLiveReload {
     export interface LiveReloadOptions {
         ignore: string,
         ghostMode: boolean,
+        tunnel: boolean,
```

Comment:
--devicesync works exactly like --LiveReload except that in addition to everything else, it syncs user actions across browsers.